services/datamanager/builtin/builtin.go

```diff
@@ -137,7 +137,7 @@ func NewBuiltIn(_ context.Context, r robot.Robot, _ config.Service, logger golog
 		collectors:                make(map[componentMethodMetadata]collectorAndConfig),
 		backgroundWorkers:         sync.WaitGroup{},
 		lock:                      sync.Mutex{},
-		syncIntervalMins:          -1,
+		syncIntervalMins:          0,
```

Comment:
Can remove the comment above `Set syncIntervalMins = -1..`. Really nice to have this implementation that only creates a syncer if enabled && syncIntervalMins > 0.0, and just enables it separately in the manual Sync() call.